widget/tabcontainer.go

```diff
@@ -163,6 +163,25 @@ func (t *TabContainer) buildTabBar(buttons []fyne.CanvasObject) *fyne.Container
 	return tabBar
 }
 
+
+// Append adds a new CanvasObject to the end of the group
+func (t *TabContainer) Append(item *TabItem) {
+	r := Renderer(t).(*tabContainerRenderer)
+	t.Items = append(t.Items, item)
+	r.tabBar.Objects = append(r.tabBar.Objects, t.makeButton(item))
+
+	t.CreateRenderer().Refresh()
```

Comment:
This is not correct - createRenderer creates a new renderer, then refreshes it. This is an orphan renderer.

Reply:
t.Refresh() should fix it

Reply:
Yup, sorry, got a wrong way here:)